### PR TITLE
TF migration docs - round 1

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/test-fairy/api-reference/upload-api.md [hunk 1]
```diff
@@ -8,6 +8,218 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
\ No newline at end of file
+### Usage
+
+[Command line uploader](https://github.com/testfairy/command-line-uploader/blob/master/testfairy-uploader.sh)
+
+[Jenkins](https://plugins.jenkins.io/TestFairy)
+
+[Gradle](https://github.com/testfairy/testfairy-gradle-plugin)
+
+Android Studio http://docs.testfairy.com/Android/Uploading_with_Android_Studio.html
```
Comment (LoryApiFortress) [inline docs/test-fairy/api-reference/upload-api.md:21]: What should I do with this link? Is not mentioned in the spreadsheet

File: docs/test-fairy/app-distribution/auto-update.md [hunk 2]
```diff
@@ -8,6 +8,79 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+- (iOS) The app is not signed with an ad-hoc or enterprise certificate.
+
+### Forcing Auto-Update
+
+Occasionally you want all testers of an app only to test the latest version released. In this case, you can use the following method to ensure users and testers cannot run older versions of the app and must upgrade to the version marked for auto-update.
+
+The classes used are:
+
+- Android - `sessionStateListener` https://docs.testfairy.com/reference/android/com/testfairy/SessionStateListener.html#SessionStateListener → `onAutoUpdateDismissed`
+- iOS - `testFairySessionStateDelagate` https://app.testfairy.com/reference/ios/Protocols/TestFairySessionStateDelegate.html → `autoUpdateDismissed`
```
Comment (LoryApiFortress) [inline docs/test-fairy/app-distribution/auto-update.md:75]: What should I do with this link?

File: docs/test-fairy/api-reference/upload-api.md [hunk 3]
```diff
@@ -8,6 +8,218 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+ <tbody>
+ <tr>
+ <td><code>api_key</code></td>
+ <td><p><small>| REQUIRED |</small></p><p>Your API application key. See https://app.testfairy.com/settings for details.</p></td>
+ </tr>
+ </tbody>
+ <tbody>
+ <tr>
+ <td><code>file</code></td>
+ <td><p><small>| REQUIRED |</small></p><p>APK (Android), AAB (Android App Bundle), IPA (iOS), or ZIP (MacOS) file data.</p></td>
```
Comment (LoryApiFortress) [inline docs/test-fairy/api-reference/upload-api.md:60]:
```suggestion
 <td><p><small>| REQUIRED |</small></p><p>Android Package Kit (APK), Android App Bundle (AAB), iOS package App Store (IPA), or ZIP (MacOS) file data.</p></td>
```

File: docs/test-fairy/api-reference/rest-api.md [hunk 4]
```diff
@@ -8,6 +8,860 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-/API/Rest_API.html
+This is the reference document for the TestFairy REST API. This API allows the developer to access and interact with TestFairy data remotely.
```
Comment (LoryApiFortress) [inline docs/test-fairy/api-reference/rest-api.md:11]:
```suggestion
In this document you can find the reference for the TestFairy REST API. This API allows the developer to access and interact with TestFairy data remotely.
```

**Review by aoleszyn — APPROVED** [review]

**Pull request merged**